creator/lib/screens/salon_application_screen.dart

```diff
@@ -13,6 +15,13 @@ class SalonApplicationScreen extends StatefulWidget {
 }
 
 class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
+  final HttpsCallable callable =
+      FirebaseFunctions.instance.httpsCallable('genericEmail');
+
+  final FirebaseAuth auth = FirebaseAuth.instance;
+
+  String emailAddress = 'kondo.matr02@gmail.com';
```

Comment:
Looks like this is not being used anywhere?

Reply:
Sorry, Forgot to erase.

Reply:
Sorry, Forgot to erase.